CvGameCoreDLL/CvGameTextMgr.cpp

```diff
@@ -14301,17 +14301,36 @@ void CvGameTextMgr::setCorporationHelpCity(CvWStringBuffer &szBuffer, Corporatio
 		}
 	}
 
-	int iNumResources = 0;
+	// int iNumResources = 0;

```

Comment:
Why is this adding commented code?

Reply:
I replaced this with the bResources section. (lines 14314-14331 of the new code) iNumResources was only use once (line 14333 of the new code) and I changed that accordingly. It doesn't change the functionality, but it was easier to include the Brazilian UP and the global resource cap this way. I didn't remove the old code and only commented it out.